cli/lib/cli.js

```diff
@@ -456,13 +446,19 @@ module.exports = {
       program.help()
     })
 
-    program
+    const handleVersion = (cmd) => {
+      return cmd
+      .option('--component <package|binary|electron|node>', 'component to report version for')
+      .action((opts, ...other) => {
+        showVersions(util.parseOpts(opts))
+      })
+    }
+
+    handleVersion(program
+    .storeOptionsAsProperties()
```

Comment:
using option `--version/-v` and using `version` as command is non longer supported and is considered "legacy".